transaction/src/manifest/ast.rs

```diff
@@ -251,7 +252,7 @@ pub enum Value {
     // Other interpreted types
     Expression(Box<Value>),
     Blob(Box<Value>),
-    NonFungibleAddress(Box<Value>),
+    NonFungibleAddress(Vec<Value>),
```

Comment:
Better to be represented as `(Box<Value>, Box<Value>)`

Reply:
Implemented.